editoast/editoast_schemas/Cargo.toml

```diff
@@ -8,8 +8,10 @@ license = "LGPL-3.0"
 derivative.workspace = true
 editoast_common.workspace = true
 enum-map.workspace = true
+geojson = "0.24.1"
```

Comment:
Just define it in the workspace Cargo.toml with a relaxed version in order to let geos use the one it likes.